src/tape_drivers/linux/sg/sg_tape.h

```diff
@@ -82,6 +82,7 @@ struct sg_data {
 	struct timeout_tape  *timeouts;            /**< Timeout table */
 	struct tc_drive_info info;                 /**< Drive information */
 	FILE*                profiler;             /**< The file pointer for profiler */
+	uint8_t              recursive_counter;    /**< Recursive counter for take dump */
```

Comment:
`int` is my preference because there is no reason to use width defined integer.

Reply:
Width defined integer must be used only for width defined field.

I believe there is no reason to fix this field as 8-bit wide. You need to use native data type, `char`, `short`, `int`, `long` or `long long`.